packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/driver_test.py

```
@@ -0,0 +1,144 @@
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('mock_data', [
+    ({'login': {'side_effect': CorelliumApiException('login error')}}),
+    ({'get_project': {'return_value': None}}),
+    ({'get_instance': {'return_value': None}}),
+    ({'create_instance': {'side_effect': CorelliumApiException('create error')}}),
+    ({'read_instance_state': {'side_effect': CorelliumApiException('read error')}}),
+])
+async def test_driver_power_on_error(mock_data):
+    project = Project('1', 'Default Project')
```

Ruff (0.8.2)

`Project` may be undefined, or defined from star imports

(F405)